pkg/api/controller.go

```diff
@@ -5470,6 +5470,13 @@ func paginationDelimiter(v *apigen.PaginationDelimiter) string {
 	return string(*v)
 }
 
+func paginationSearch(v *apigen.SearchString) string {
```

Comment:
```suggestion
func search(v *apigen.SearchString) string {
```
I don't see what this has to do with pagination.